### PR TITLE
Init CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jdcaperon**

### CODE REVIEW

**Comment by jdcaperon** [comment]
@nicklambourne what else should we add to this?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+You can contribute code by forking the repository, making your changes and creating a pull request. It's encouraged to create an issue for the changes you plan on making in your pull request and you should mention the IDs of any issues you work on in your pull request description.
+
+Want help with creating a pull request? No problem!
+Email us at contact@uqcs.org.au for help, or ask in [Slack](https://slack.uqcs.org.au/).
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## The Contribution Process
+
+1. Find an issue that you want to take on and [assign yourself to it](https://help.github.com/en/github/managing-your-work-on-github/assigning-issues-and-pull-requests-to-other-github-users), or c[reate an issue](https://help.github.com/en/github/managing-your-work-on-github/creating-an-issue) for the work you are about to do.
```
Comment (nicklambourne) [inline CONTRIBUTING.md]: c[reate an issue]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+2. Create a branch to begin work on either using the [GitHub UI](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-and-deleting-branches-within-your-repository), or via your [command line](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging).
+3. Checkout the branch and work on your issue.
+4. When you are finished on your feature/issue, push it to the remote repository and [create your pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request).
+5. [Request a review](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/requesting-a-pull-request-review) from one of the reviewers listed in the reviewers section of this document.
+6. When your pull request is reviewed and approved, click `Squash and Merge`, clear the commits from below the merge title, then confirm.
```
Comment (nicklambourne) [inline CONTRIBUTING.md]: Is S&M the default?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+## Reviewers
+
+When merging a pull request, make sure you have received a review from one of the following people:
+
+* [@jam](https://github.com/jdcaperon)
```
Comment (nicklambourne) [inline CONTRIBUTING.md]: You can put me down for this.

File: CONTRIBUTING.md (hunk 2 above)
Comment (jdcaperon) [reply CONTRIBUTING.md]: It is now :)

File: CONTRIBUTING.md (hunk 3 above)
Comment (jdcaperon) [reply CONTRIBUTING.md]: 😍

**Pull request merged**